pkgs/applications/audio/musikcube/default.nix

```diff
@@ -6,6 +6,7 @@
 , fetchFromGitHub
 , fetchpatch
 , ffmpeg
+, gnutls
```

Comment:
ℹ️ Compilation failed with the following error without `gnutls` being in the `buildInputs`:

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LIBGNUTLS
    linked by target "server" in directory /tmp/nix-build-musikcube-0.97.0.drv-0/source/src/plugins/server
```